R/restore.R

```diff
@@ -584,6 +584,16 @@ installPkg <- function(pkgRecord,
       quiet <- isTRUE(packrat::opts$quiet.package.installation())
       install_local_path(path = pkgSrc, reload = FALSE,
                          dependencies = FALSE, quick = TRUE, quiet = quiet)
+      # if we just installed a binary package, check that it can be loaded
+      # (source packages are checked by default on install)
+      if (identical(pkgType, "binary")) {
+        status <- renv$catch(renv$renv_install_test(pkgRecord$name))
+        if (inherits(status, "error")) {
+          message("FAILED")
```

Comment:
Does this message look out of place in context? I.e. does it convey anything additional that `renv$abort` doesn't convey?

Reply:
Here is the full output from the `packrat::restore()` performed by the Docker build. The "FAILED" before `stop` is consistent with other exiting conditions throughout the restore.

Thoughts?

```
#15 [installation 7/8] RUN R -s -e "packrat::restore()"
#15 0.652 Installing coda (0.19-4) ...
#15 2.423 	OK (downloaded binary)
#15 2.423 Installing packrat (0.9.1-1) ...
#15 7.818 	OK (built source)
#15 7.819 Installing rjags (4-14) ...
#15 9.056 FAILED
#15 9.057 Error: error testing if 'rjags' can be loaded [error code 1]
#15 9.057 /opt/R/4.3.0/lib/R/bin/R --vanilla -s -f '/tmp/RtmpA18AdZ/renv-install-74536342'
#15 9.057 ================================================================================
#15 9.057
#15 9.057 Loading required package: coda
#15 9.057 Error: package or namespace load failed for ‘rjags’:
#15 9.057  .onLoad failed in loadNamespace() for 'rjags', details:
#15 9.057   call: dyn.load(file, DLLpath = DLLpath, ...)
#15 9.057   error: unable to load shared object '/content/packrat/lib/x86_64-pc-linux-gnu/4.3.0/rjags/libs/rjags.so':
#15 9.057   libjags.so.4: cannot open shared object file: No such file or directory
#15 9.057 Execution halted
#15 9.057
#15 9.057 Execution halted
#15 ERROR: process "/bin/sh -c R -s -e \"packrat::restore()\"" did not complete successfully: exit code: 1
```